client/a8c-for-agencies/sections/marketplace/products-overview-v2/index.tsx

```diff
@@ -196,7 +196,7 @@ export function ProductsOverviewV2( {
 			<div
 				className="products-overview-v2__action-panel-wrapper"
 				ref={ actionPanelRef }
-				style={ { top: actionPanelStickyTopOffset } }
+				style={ { top: actionPanelStickyTopOffset, zIndex: 20 } }
```

Comment:
Why do we need zIndex here?

Reply:
For some reason, the disabled state of the card overlaps the sticky action panel. I have to increase it to 20.